src/Permissions/Backend/PermissionManager.vala

```diff
@@ -36,95 +36,59 @@ public class Permissions.Backend.PermissionManager {
-        insert ("sockets=x11", new PermissionDescription (
-            _("Graphical output"),
-            _("App may be able to show graphical windows"),
-            "video-display"
-        ));
-        //  insert ("sockets=fallback-x11", new PermissionDescription (
-        //      _("Access X11 windowing system (as fallback)"),
-        //      _("App is able to show graphical windows"),
-        //      "video-display"
-        //  ));
-        //  insert ("sockets=wayland", new PermissionDescription (
-        //      _("Access Wayland windowing system"),
-        //      _("App is able to show graphical windows"),
-        //      "video-display"
-        //  ));
-        insert ("sockets=pulseaudio", new PermissionDescription (
-            _("Sounds"),
-            _("App may be able to play sounds"),
-            "preferences-desktop-sound"
-        ));
-        insert ("sockets=system-bus", new PermissionDescription (
-            _("D-Bus system bus"),
-            _("App may be able to talk to, control and read information from system components via D-Bus"),
-            "internet-chat"
-        ));
-        insert ("sockets=session-bus", new PermissionDescription (
-            _("D-Bus session bus"),
-            _("App may be able to talk to, control and read information from other apps via D-Bus"),
-            "internet-chat"
-        ));
-        insert ("sockets=ssh-auth", new PermissionDescription (
-            _("Secure Shell agent"),
-            _("App may be able to access other devices on the network via SSH"),
-            "utilities-terminal"
-        ));
-        insert ("sockets=cups", new PermissionDescription (
-            _("Print"),
-            _("App may be able to access printers"),
-            "printer-printing"
+
+        insert ("filesystems=home", new PermissionDescription (
+            _("Home Folder"),
+            _("Access your entire home folder"),
+            "user-home"
         ));
-        insert ("devices=dri", new PermissionDescription (
-            _("GPU acceleration"),
-            _("App may be able to accelerate graphical output"),
-            "applications-graphics"
+
+        insert ("filesystems=host", new PermissionDescription (
+            _("System Folders"),
+            _("Access system folders, not including the operating system or system internals"),
+            "drive-harddisk"
         ));
+
         insert ("devices=all", new PermissionDescription (
             _("Devices"),
-            _("App may be able to access devices like webcams"),
+            _("Access devices like webcams and microphones"),
             "accessories-camera"
         ));
-        insert ("filesystems=host", new PermissionDescription (
-            _("System directories"),
-            _("App may be able to access system directories"),
-            "drive-harddisk"
+
+        insert ("sockets=pulseaudio", new PermissionDescription (
+            _("Sounds"),
+            _("Play sounds"),
```

Comment:
Do we know if this socket also allows recording other apps' sounds or the mic?

Reply:
The flatpak docs say "Play sound with pulseaudio", so I think that implies no, but we should test it

Reply:
Okay it looks like apps can still play sound directly via alsa with this disabled, so this is bullshit